docs/JSON-RPC.md

```diff
@@ -471,3 +492,43 @@ Parameters:
 | params | object | No parameters (empty object). |
 
 
+### jamulusclient/recorderState
+
+Emitted when the client is connected to a server whose recorder state changes.
+
+Parameters:
+
+| Name | Type | Description |
+| --- | --- | --- |
+| params.state | number | The recorder state |
+
+
+### jamulusclient/serverInfoReceived
+
+Emitted when a server info is received.
+
+Parameters:
+
+| Name | Type | Description |
+| --- | --- | --- |
+| params.address | string | The server socket address |
+| params.pingtime | number | The round-trip ping time in ms |
+| params.numClients | number | The quantity of clients connected to the server |
```

Comment:
```suggestion
| params.numClients | number | The number of clients connected to the server. |
```

Perhaps there's a notion that the definition of something shouldn't use the same word. Maybe. But the only existing similar usage on this page is result.clients[*].channels, and it says number rather than quantity.

Reply:
(And the aim is to contextualise "number of clients" rather than explain "number", "of" or "clients".)

Reply:
Sorry! I tend to use the word, "quantity" in docs to avoid ambiguity. Feel free to use the language with which you feel most comfortable.